docs/sphinx/developers/cpp/tutorial.txt

```diff
@@ -66,7 +71,26 @@ themselves are more of an implementation detail at present.
    :start-after: read-example-start
    :end-before: read-example-end
 
-Full example source: :download:`metadata-formatreader.cpp <examples/metadata-formatreader.cpp>`
+If implementing a reader, it is fairly typical to set the basic image
```

Comment:
We do use it ourselves, but there is no obligation to do so.  While it is certainly convenient, a reader implementer is free to do their own thing.  However, this section isn't about that, it's about conversion between metadata forms, so I was trying to avoid going into specifics at this point in the tutorial; this is just an example of where it could be used.